v0/destinations/clickup/util.js

```diff
@@ -0,0 +1,382 @@
+const _ = require("lodash");
+const { httpGET } = require("../../../adapters/network");
+const {
+  processAxiosResponse
+} = require("../../../adapters/utils/networkUtils");
+const {
+  isEmpty,
+  getHashFromArray,
+  formatTimeStamp,
+  CustomError
+} = require("../../util");
+const { getCustomFieldsEndPoint } = require("./config");
+
+/**
+ * Validates priority
+ * @param {*} priority
+ */
+const validatePriority = priority => {
+  if (
+    (priority &&
+      !(Number.isInteger(priority) && priority >= 1 && priority <= 4)) ||
+    priority === 0
+  ) {
+    throw new CustomError(
+      "Invalid Priority. Value must be Integer and in range [1,4]",
+      400
+    );
+  }
+};
+
+/**
+ * Validates phone with country code
+ * @param {*} phone
+ */
+const validatePhoneWithCountryCode = phone => {
+  const regex = /^\+(?:[{0-9] ?){6,14}[0-9]$/;
+  if (!regex.test(phone)) {
+    throw new CustomError("Invalid Phone Number", 400);
+  }
+};
+
+/**
+ * Validates email
+ * @param {*} email
+ */
+const validateEmail = email => {
+  const regex = /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/;
+  if (!regex.test(email)) {
+    throw new CustomError("Invalid Email", 400);
```

Comment:
Please format the error message for all the error messages in all the files for clickup cloud mode implementation

Reply:
Updated all error messages